spec/models/comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject { User.new(Name: 'Tom', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.') }
+
+  before { subject.save }
+
+  before(:each) do
+    @post = Post.create(AuthorId: subject.id, Title: 'check post',
+                        Text: 'Unit test for the method of most_recent_posts')
+    @comment = Comment.create(AuthorId: subject.id, PostId: @post.id,
+                              Text: 'Unit test for the method of most_recent_comments')
+  end
+
+  it 'Text should match with the input text' do
+    text = @comment.Text
+
+    expect(text).to eq('Unit test for the method of most_recent_comments')
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure to fix these errors so the rspec tests are passing 👇🏼 
![Screenshot_20230128_204624](https://user-images.githubusercontent.com/25227022/215287956-d83788be-3336-4634-b623-aee77d89bb86.png)